src/add/observable/combineLatest.ts

```diff
@@ -1,6 +1,4 @@
 import {Observable} from '../../Observable';
 import {combineLatestStatic} from '../../operator/combineLatest';
 
-Observable.combineLatest = combineLatestStatic;
-
-export var _void: void;
```

Comment:
Yes, that's right. I'll revert the `_void: void` removals and re-exports for the `observable` patches.